### ISSUE TITLE
Cookie expiry should clear the cookie in native app

### ISSUE THREAD

**Issue by vegetableman** [a85/POSTMan-Chrome-Extension#2404]
I also have this problem. I can't clear the cookie in the mac app version. (It works in the chrome app version)

Login API response header

```
Set-Cookie →AuthSession=YWRtaW5AYWRtaW4uY29tOjU3RkRGQTY1OsDcEhZBHW94xEgUFGX9yn_PokWj; path=/; httponly
```

Cookie viewed from `Manage Cookies` section.

```
AuthSession=YWRtaW5AYWRtaW4uY29tOjU3RkRGQTY1OsDcEhZBHW94xEgUFGX9yn_PokWj; path=/; domain=datana; HttpOnly; Expires=Tue Jan 19 2038 03:14:07 GMT+0700 (ICT);
```

Logout API response header

```
Set-Cookie →AuthSession=; path=/; expires=Thu, 01 Jan 1970 00:00:00 GMT; httponly
```

cc @siwatpru

**Comment by vegetableman** [comment]
This issue has been patched in our internal build. Will keep you posted as to the release version for it ^_^

**Comment by siwatpru** [comment]
Thank you :)

**Comment by simplesmiler** [comment]
Same thing happens on Windows client 4.8.1.
Step by step: http://imgur.com/a/btmTW

**Comment by vegetableman** [comment]
The issue has been fixed in the recent release `4.8.3`.  (Reopening it, so that you guys could verify and let me know)

**Comment by vegetableman** [comment]
Closing this issue. Let me know if the issue still exists ^_^.
